introduction/getting-started.qmd

```diff
@@ -104,23 +101,23 @@ Every step in a research design can be declared using one of the `declare_*` fun
 
 : Declaration functions in DeclareDesign {#tbl-declarationfunctions} 
 
-We use the `+` operator to build from elements of a design to a design. @def-ch4num1 shows the format of most declarations throughout the book. This declaration represents a two-arm randomized experiment with 100 units from which we aim to estimate the average treatment effect.
+We use the `+` operator to build from elements of a design to a design. @def-ch4num1 (see @fig-designannotated) shows the format of most declarations throughout the book. This declaration represents a two-arm randomized experiment with 100 units from which we aim to estimate the average treatment effect.
+
+
+## Diagnosis {#diagnose-design-function}
+
+Diagnosis is the process of simulating the design many times and calculating summary statistics about the design that describe its properties, which we call diagnosands. Once a design is declared, diagnosis is as simple as using the `diagnose_design` function on it.
+
 
 ::: {#def-ch4num1 .declaration} 
```

Comment:
This declaration should be in the section on declaration, it appears to now be in the section on Diagnosis. Would you review the book again on this? See https://book.declaredesign.org/introduction/getting-started.html - it looks like its placement is already correct.